packages/neuron-ui/src/components/SendFieldset/index.tsx

```diff
@@ -75,16 +65,35 @@ const SendFieldset = ({
   return (
     <div className={styles.container}>
       <TextField
-        className={styles.addressField}
-        label={t('send.address')}
+        label={
+          <div className={styles['remove-label']}>
+            <div>{t('send.address')}</div>
+            {isRemoveBtnShow && (
```

Comment:
Use ternary to render elements explicitly, or unexpected items may be rendered accidentally along with development